src/bin/owi.ml

```diff
@@ -37,6 +37,19 @@ let path_conv = Arg.conv (Fpath.of_string, Fpath.pp)
 
 let solver_conv = Arg.conv (Smtml.Solver_type.of_string, Smtml.Solver_type.pp)
 
+let model_format_conv =
+  let of_string s =
+    match String.lowercase_ascii s with
+    | "scfg" -> Ok Cmd_utils.Scfg
+    | "json" -> Ok Json
+    | _ -> Error (`Msg "Expected \"json\" or \"scfg\" but got %s\n")
```

Comment:
Two remarks, instead of `"hello"` it is possible to use the `{|` and `|}` delimiters in OCaml: `{|hello|}`, moreover, you can "customize" these separators, for instance, you could use `{html|` and `|html}` and write: `{html|<h1>My title</h1>|html}`.

Here, it would allow to write: `Msg {|Expected "json" or "scfg" but got %s\n|}`.

On a related note, there is a very useful function `Fmt.error_msg`, you can use it like this: `Fmt.error_message {|Expected "json" or "scfg" but got %s\n|}`.

(Also, I'm not sure the code is currently doing what you expect, I see a `%s` but no `Fmt.str` and no string is given either, using Fmt.error_msg will make it easy to solve).